docs/src/install.md

```diff
@@ -0,0 +1,14 @@
+# Build freenet from source On Linux:
+
+There is a single line command to build all freenet on Linux.
```

Comment:
This is not "building freenet", is building and running the email-app locally. Let's be more specific about that.

"If you want to run the Freenet Mail App locally you can follow this steps". This also does a bunch of things that don't have to do with that directly.

Also this is redundant with

Reply:
fixed!